features/in_onderzoek.feature

```diff
@@ -921,45 +1137,70 @@ Functionaliteit: in onderzoek
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550001 |
-      En de persoon met burgerservicenummer '555550001' heeft de volgende 'naam' gegevens
-      | naam                          | waarde                  |
-      | voornamen (02.10)             | Piet                    |
-      | geslachtsnaam (02.40)         | Groenen                 |
-      | aanduidingNaamgebruik (61.10) | <aanduidingNaamgebruik> |
-      En de persoon heeft een partner met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 050200   |
-      | datumIngangOnderzoek          | 20220307 |
+      Gegeven de persoon met burgerservicenummer '000000590' heeft de volgende gegevens
+      | naam                           | waarde                  |
+      | voornamen (02.10)              | Piet                    |
+      | geslachtsnaam (02.40)          | Groenen                 |
+      | aanduiding naamgebruik (61.10) | <aanduidingNaamgebruik> |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                            | waarde    |
+      | burgerservicenummer (01.20)     | 000000607 |
+      | aanduiding in onderzoek (83.10) | <waarde>  |
+      | datum ingang onderzoek (83.20)  | 20220307  |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000590                       |
+      | fields              | burgerservicenummer,adressering.inOnderzoek.aanhef,adressering.inOnderzoek.aanschrijfwijze,adressering.inOnderzoek.gebruikInLopendeTekst |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                                                            | waarde         |
+      | burgerservicenummer                                             | 000000590      |
+      | adressering.inOnderzoek.aanhef                                  | <in onderzoek> |
+      | adressering.inOnderzoek.aanschrijfwijze                         | <in onderzoek> |
+      | adressering.inOnderzoek.gebruikInLopendeTekst			| <in onderzoek> |
+      | adressering.inOnderzoek.datumIngangOnderzoekPartner.type        | <datum type>   |
+      | adressering.inOnderzoek.datumIngangOnderzoekPartner.datum       | <datum datum>  |
+      | adressering.inOnderzoek.datumIngangOnderzoekPartner.langFormaat | <datum lang>   |
+
+      Voorbeelden:
+      | omschrijving                                            | waarde | aanduidingNaamgebruik | in onderzoek | datum type | datum datum | datum lang   |
```

Comment:
Neem voor de duidelijkheid ook een voorbeeld op waarbij 'aanduidingNaamgebruik' de waarde 'E' heeft.

Reply:
Is gedaan.